modules/apps/journal/journal-web/src/main/resources/META-INF/resources/edit_article.jsp

```diff
@@ -145,6 +145,15 @@ journalEditArticleDisplayContext.setViewAttributes();
 					<div class="c-gap-3 form-group-sm journal-article-button-row mb-0 tbar-section text-right">
 						<c:choose>
 							<c:when test='<%= FeatureFlagManagerUtil.isEnabled("LPS-141392") %>'>
+								<div class="align-items-center d-none mx-3 small text-danger" id="<portlet:namespace />lockErrorIndicator">
+									<liferay-ui:message key="alert-helper-error" />
+
+									<clay:icon
+										cssClass="ml-2"
```

Comment:
please add class `mt-0 ` so the icon is properly aligned

Reply:
I force pushed the requested changes